services/director-v2/tests/conftest.py

```diff
@@ -265,7 +259,7 @@ def user_db(postgres_db: sa.engine.Engine, user_id: PositiveInt) -> Dict:
     with postgres_db.connect() as con:
         # removes all users before continuing
         con.execute(users.delete())
-        result = con.execute(
+        result1 = con.execute(
```

Comment:
no need for any result variable here

Reply:
actually that might have been a typo ;)... will check